.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Semantic Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
          GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

GITHUB_TOKEN is not an allowed secret name, so I had to name it GH_TOKEN which is also supported by semantic-release.

![2021-05-26 20_00_16-Actions secrets and 8 more pages - Personal - Microsoft​ Edge](https://user-images.githubusercontent.com/6810177/119709002-03356900-be5d-11eb-8836-057e16aadcd6.png)

Reply:
Oh, interesting! Is it only in this repo? Because in my test projects, it wasn't a problem.

Reply:
it's an org secret, so there is no repo context